src/features/MentorPage/components/MentorList/index.tsx

```diff
@@ -1,7 +1,7 @@
 import styled, { css } from 'styled-components';
 import { spacing } from '@/components/variables';
 import ListCard from './MentorCard/List';
-import { Mentor } from '../mentorPageApi';
+import { Mentor } from '../../mentorPageApi';
```

Comment:
I'd prefer `@/features/MentorPage/mentorPageApi` to `../../` when "going up" the directory tree.